CMakeLists.txt

```diff
@@ -1,53 +1,118 @@
-cmake_minimum_required(VERSION 2.6 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Is cmake version 3.9 really necessary ?
If no, what is the minimum version we should use ?
If yae, for which feature ?

Reply:
I don't know exactly for safety I use the one used by the lasts ROOT. Anyway i think even ilcsoft download an newer version. 2.6 is very ugly !

Reply:
CMake_version = "3.15.5" in the ilcinstall in github so 3.9 should be fine I would like 3.12 to be safe

Reply:
Fine, I 've misread the ilcsoft default.